src/components/TransactionDialog/TransactionDialogContent.tsx

```diff
@@ -64,6 +65,34 @@ const TransactionSuccessView: FC<{
             sourceAsset?.symbol ?? route.source_asset_denom
           } from ${sourceChain.prettyName} to ${destinationChain.prettyName}`}
       </p>
+      <div className="flex-1 space-y-6 w-full">
```

Comment:
we may want to start considering when to pull these things out into their own components to make it easier at a glance to see what is responsible for what.

Reply:
Add a linear issue for this @thal0x